### DISCUSSION TITLE
(SUNBIRD-SAAS) Content APIs version

### DISCUSSION THREAD

**Discussion by chaitanyakole** [Sunbird-Ed/Community#430] [Documentation]
I tried to create a content through postman using this Content API and was able to create successfully.

Checked the content created through postman on portal and there able to see the content, but when I'm trying to send the content to review through postman, this API is giving me the following error:
 **_"Error! Missing One or More Required Fields in Object. | [Either 'body' or 'artifactUrl' are required for processing of ECML content!"_** 

I tried to follow this http://docs.sunbird.org/latest/apis/content/index.html#operation/Submit%20Content%20for%20Review and still facing the error.

I tried to create a content on portal and was able to send it for review successfully, but noticed one thing, i.e, on portal v3 version is being used and in the document on v1 version API's are available for send for review. And if I try to use v3 version API through postman, then it's giving me error:
 _**"no route and no API found with those values"**_

 To: @vinukumar-vs @rajeevsathish  @rajeshkumaravel

CC: @@nidhikachhawa24 @Baljeetsingh

**Comment by Jayaprakash8887** [comment]
All API mapping to respective services can be found in https://github.com/project-sunbird/sunbird-devops/blob/release-5.2.0/ansible/roles/kong-api/defaults/main.yml

**Comment by Jayaprakash8887** [comment]
"Error! Missing One or More Required Fields in Object. | [Either 'body' or 'artifactUrl' are required for processing of ECML content!" is a valid error message. This means that there is no "content" source file for the content you are trying to review. Please check if 'body' or 'artifactUrl' property exists when you read the content.

**Comment by vinukumar-vs** [comment]
@chaitanyakole Please refer the below postman collection for Sunbird Knowlg BB API's
https://www.postman.com/sunbird-building-blocks/workspace/sunbird-knowlg/collection/25463377-2979d790-f956-4d73-b080-9af022721fbf?ctx=documentation

**sample:**
{{host}} => **https://dev.sunbirded.org/api** 
API path should be - https://dev.sunbirded.org/api/content/v2/create
